pages/profile/[username].tsx

```diff
@@ -101,7 +117,11 @@ export default function Profile(props: PageProps) {
                     {data?.allow_anon_dm ? (
                         <>
                             <div className="flex flex-grow bg-gray05 rounded-xl py-1 px-2">
-                                <div contentEditable onInput={(e) => setMessage(e.currentTarget.textContent)} className="bg-transparent min-h-[24px] w-full" placeholder="Send a message..." />
+                                <textarea ref={textAreaRef} rows={1} onChange={(e) => {
+                                    setMessage(e.target.value)
+                                }} className="bg-transparent min-h-[24px] w-full"
+                                    style={{ resize: 'none' }}
+                                    placeholder="Send a message..." />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `contentEditable` div has been replaced with a `textarea` element. This change allows for better control over the input and its properties. The `onChange` event handler has also been updated to set the message state according to the textarea value. This is a good practice for handling user input in React.
```diff
- <div contentEditable onInput={(e) => setMessage(e.currentTarget.textContent)} className="bg-transparent min-h-[24px] w-full" placeholder="Send a message..." />
+ <textarea ref={textAreaRef} rows={1} onChange={(e) => {
+     setMessage(e.target.value)
+ }} className="bg-transparent min-h-[24px] w-full"
+     style={{ resize: 'none' }}
+     placeholder="Send a message..." />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->